molgenis-core-ui/src/main/resources/templates/view-thememanager.ftl

```diff
@@ -3,60 +3,56 @@
 
 <#assign css=[]>
 <#assign js=["thememanager.js"]>
+<#assign version = 2>
 
-<@header css js/>
+<@header css js version/>
+<div class="row" id="thememanager">
+    <div class="col-md-12">
+        <legend>Theme Manager</legend>
```

Comment:
Add comment to explain that this functions as vue template for the thememanager.js

Reply:
Will do!